spec/support/features/doc_auth_helper.rb

```diff
@@ -14,8 +16,8 @@ def session_from_completed_flow_steps(finished_step)
 
   def fill_out_ssn_form_with_duplicate_ssn
     diff_user = create(:user)
-    create(:profile, pii: { ssn: '123-45-6666' }, user: diff_user)
-    fill_in 'doc_auth_ssn', with: '123-45-6666'
+    create(:profile, pii: { ssn: '800-66-6666' }, user: diff_user)
```

Comment:
**question:** is this supposed to start with `900`?

Reply:
this is a weird test that I think was written to fail on duplicate SSNs, but we don't do that, so it fails for another reason so the test passes? 🤷🏼 

I can probably set any test using this as a skip instead

Reply:
Yeah, it looks like we dropped that in https://github.com/18F/identity-idp/pull/3647, so we could probably even delete these tests...

Reply:
Yeah, let's pull these

Reply:
Love to delete feature tests. Dropped them in 4129bed99